model/index.js

```diff
@@ -165,13 +166,13 @@ module.exports = class Model extends EventEmitter {
   handleResponse(response, callback) {
     let data = {};
     try {
-      data = JSON.parse(response.body || '{}');
+      data = typeof response.data === 'object' ? response.data : JSON.parse(response.data || '{}');
```

Comment:
tenary can sometimes be hard to read sometimes

Reply:
I have changed this to an if/else statement now

Reply:
it's so subjective these things but glad we agree 🙂